### ISSUE TITLE
plex shows two instances of mpv-plex-shim, only one running

### ISSUE THREAD

**Issue by geeohgeegeeoh** [iwalton3/plex-mpv-shim#49]
![Screen Shot 2021-09-20 at 11 10 37 am](https://user-images.githubusercontent.com/6101116/133949266-fe57028c-3d30-4d70-b52a-fb4b3bf09b7d.png)
I always see two instances of the shim in my sources list.

client is plex web player on OSX/Chrome

mpv-plex-shim is running via an mpv instance (not the libmpv binding) on Ubuntu/RPi4 over ethernet.

**Comment by geeohgeegeeoh** [comment]
here's my runstate: one python3 invocation of the shim, one instance of mpv running.
```
ggm@magicpudding:~$ ps ax|grep mpv
  49736 pts/1    Sl+   13:25 /usr/bin/python3 /usr/local/bin/plex-mpv-shim
  49743 pts/1    SLl+  14:53 mpv --input-default-bindings=yes --input-vo-keyboard=yes --input-media-keys=yes --include=/home/ggm/.config/plex-mpv-shim/mpv.conf --input-conf=/home/ggm/.config/plex-mpv-shim/input.conf --player-operation-mode=cplayer --idle=yes --input-ipc-server=/tmp/mpv193076854370656 --input-terminal=no --terminal=no
 249823 pts/0    S+     0:00 grep --color=auto mpv
ggm@magicpudding:~$
```

**Comment by LAP87** [comment]
Same issue here, Windows 10, doesn't matter which browser.

**Comment by iwalton3** [comment]
Neat... Do both of them work? Is the response from the Plex server (Devtools/F12 > Network) giving a duplicate response or are they different in some way?

I know historically the application has had issues tearing down old instances when you quit it, but as shown by the `ps` listing this issue it not that.

**Comment by geeohgeegeeoh** [comment]
> Neat... Do both of them work? 

Nope. One of them looks to behave as a Duff connection. When I don't have debug running

>Is the response from the Plex server (Devtools/F12 > Network) giving a duplicate response or are they different in some way?

I've repeat tested, and with debug running, unfortunately BOTH OF THEM WORK :-( So this is a timing problem which doesn't show, when I run debug mode.
